src/docs/user/references/ApplicationConnectors.adoc

```diff
@@ -78,3 +78,772 @@ The Informatica workflow consists of a list of variables described in the follow
+| Yes
+| Boolean
+| e.g. `false`
+| `SYSPARM_SUPPRESS_AUTO_SYS_FIELD`
+| True to suppress auto generation of system fields
+| Yes
+| Boolean
+| e.g. `false`
+| `SYSPARM_QUERY_NO_DOMAIN`
+| True to access data across domains if authorized
+| Yes
+| Boolean
+| e.g. `false`
+| `SSL_DISABLE_CHECK`
+| If true, disables SSL certificate verification
+| No
+| Boolean
+| e.g. true
+| `DEBUG`
+| If true, activates the debug mode which prints messages to the terminal.
+| No
+| Boolean
+| e.g. `true`
+|===
+
+==== ServiceNow to ProActive
+
+Once the application is installed, you are ready to integrate ProActive in your ServiceNow pipelines.
+
+We will go over the core application files in the application to explain how to use them.
+
+===== Script Includes
+
+====== Overview
+
+As defined by ServiceNow, Script Includes are reusable server-side scripts logic that define a function or a class.
+Script Includes execute their script logic only when explicitly called by other scripts anywhere in the application where scripting is permitted.
+
+It can be in another Script Include, In Workflow Activities, Flow Actions, UI elements ...
+
+====== Usage
+
+Calling a Script Include where scripting is enabled is very easy. You just need to create an instance of the Script Include and execute its functions.
+
+Here is an example where the Script Include named SchedulerClient is called in a Flow's Action to submit a job:
+
+[[submit_from_catalog_script_code]]
+[source,javascript]
+----
+(function execute(inputs, outputs) {
+
+    var schedulerClient = new SchedulerClient();
+    var response = schedulerClient.submitJobFromCatalog(inputs.proactive_url, inputs.session_id, inputs.bucket_name, inputs.workflow_name, inputs.variables, inputs.generic_info);
+
+    outputs.job_id = response.id;
+    outputs.readable_name = response.readableName;
+
+})(inputs, outputs);
+----
+
+====== ProActive's application Script Includes
+
+There are 3 script includes in the ProActive application:
+
+*SchedulerClient*: A client that makes REST calls to a ProActive's Scheduler.
+Call this Script Include to make HTTP requests to the Scheduler of a ProActive's server.
+
+Here is the list of its current functions.
+
+TIP: The REST API documentation for our try platform is available at https://try.activeeon.com/doc/rest/
+
+[cols="1,1,1,1"]
+|===
+|Function name | Action | Inputs | Output
+
+|getSessionId
+|Executes a POST request to login and retrieve the sessionId of a ProActive user
+a|
+* proActiveUrl: Root URL of ProActive server
+* username: Name of user
+* password: Password of user
+|The session id associated to the user
+
+|restartInErrorTasks
+|Executes a PUT request to restart all tasks in error in the job represented by a job id
+a|
+* jobId: Id of the job to restart in error tasks
+|True if success, false if not
+
+|submitJobFromCatalog
+|Submits a workflow to the scheduler from a workflow URL, creating a new job resource
```

Comment:
Submits a workflow to the scheduler --> Submits a job to the scheduler
Could you give an example of workflow URL?

Reply:
Im going to rephrase into `Submits a workflow to the scheduler from the catalog, creating a new job resource`
as it is confusing.

You cannot actually provide a URL of a proactive workflow. Instead you specify a bucket and a workflow name that will be fetched from the catalog.